tests/integration/testdata/invoke/cdk/python/aws-lambda-function/aws_lambda_function/aws_lambda_function_stack.py

```diff
@@ -85,9 +102,30 @@ def __init__(self, scope: cdk.Construct, construct_id: str, **kwargs) -> None:
             runtime=_lambda.Runtime.PYTHON_3_8,
             code=_lambda.Code.from_asset("./lambda_code"),
             environment={
-                "MyRuntimeVersion" : my_runtime_version_parameter.value_as_string,
-                "EmptyDefaultParameter" : empty_default_parameter.value_as_string
+                "MyRuntimeVersion": my_runtime_version_parameter.value_as_string,
+                "EmptyDefaultParameter": empty_default_parameter.value_as_string
             },
             handler="app.parameter_echo_handler"
         )
 
+        _lambda.DockerImageFunction(
+            scope=self,
+            id="lambda-docker-function",
+            function_name="DockerImageFunction",
+
+            code=_lambda.DockerImageCode.from_image_asset(
+                "./docker_lambda_code",
+                cmd=['app.get'],
+                entrypoint=["/lambda-entrypoint.sh"],
+                file='Dockerfile',
+            ),
+        )
+
+        aws_lambda_python.PythonFunction(
+            scope=self,
+            id="python-function-construct",
+            runtime=_lambda.Runtime.PYTHON_3_8,
+            index="app.py",
+            entry="./lambda_code",
+            handler="handler"
+        )
```

Comment:
does the PythonFunction have any difference from the Function above with id `helloworld-serverless-functio`?

Reply:
There are some small differences in behaviour between the constructs. These language-specific Lambda Function constructs also build the source code and the generated Cloud Assembly is a little different. I wanted to include this here for completeness.